lesson14/main.js

```diff
@@ -0,0 +1,122 @@
+const openButton = document.getElementById("js-open-modal-button");
+const modal = document.getElementById("js-modal");
+const closeButton = document.getElementById("js-close-button");
+const overlay = document.getElementById("js-overlay");
+const openListButton = document.getElementById("js-open-list-button")
+const ul = document.getElementById("js-ul");
+const input =document.getElementById("js-number-input")
```

Comment:
nits
Since there are often multiple input elements, you might want to make the variable names unique.
```suggestion
const numberInput =document.getElementById("js-number-input")
```

Reply:
Changed variable name,
please see the commit
10d6a13925dbfcfc0fd346260585132aa1dd91ed